### PR TITLE
Acrolinx

### PR DESCRIPTION
**Pull request by gewarren**
---

#### Internal previews

<details><summary><strong>Toggle expand/collapse</strong></summary><br/>

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/azure/sdk/unit-testing-mocking.md](https://github.com/dotnet/docs/blob/88403af672427f61cad73a91ec9b7df470912ea4/docs/azure/sdk/unit-testing-mocking.md) | [Unit testing and mocking with the Azure SDK for .NET](https://review.learn.microsoft.com/en-us/dotnet/azure/sdk/unit-testing-mocking?branch=pr-en-us-36528) |
| [docs/core/compatibility/aspnet-core/8.0/trimmode-full.md](https://github.com/dotnet/docs/blob/88403af672427f61cad73a91ec9b7df470912ea4/docs/core/compatibility/aspnet-core/8.0/trimmode-full.md) | ["Breaking change: TrimMode defaults to full for Web SDK projects"](https://review.learn.microsoft.com/en-us/dotnet/core/compatibility/aspnet-core/8.0/trimmode-full?branch=pr-en-us-36528) |
| [docs/core/docker/build-container.md](https://github.com/dotnet/docs/blob/88403af672427f61cad73a91ec9b7df470912ea4/docs/core/docker/build-container.md) | [Tutorial: Containerize a .NET app](https://review.learn.microsoft.com/en-us/dotnet/core/docker/build-container?branch=pr-en-us-36528) |
| [docs/core/docker/introduction.md](https://github.com/dotnet/docs/blob/88403af672427f61cad73a91ec9b7df470912ea4/docs/core/docker/introduction.md) | [docs/core/docker/introduction](https://review.learn.microsoft.com/en-us/dotnet/core/docker/introduction?branch=pr-en-us-36528) |
| [docs/core/docker/publish-as-container.md](https://github.com/dotnet/docs/blob/88403af672427f61cad73a91ec9b7df470912ea4/docs/core/docker/publish-as-container.md) | [Containerize a .NET app with dotnet publish](https://review.learn.microsoft.com/en-us/dotnet/core/docker/publish-as-container?branch=pr-en-us-36528) |
| [docs/core/tools/custom-templates.md](https://github.com/dotnet/docs/blob/88403af672427f61cad73a91ec9b7df470912ea4/docs/core/tools/custom-templates.md) | [Custom templates for dotnet new](https://review.learn.microsoft.com/en-us/dotnet/core/tools/custom-templates?branch=pr-en-us-36528) |
| [docs/core/tools/dotnet-new-details.md](https://github.com/dotnet/docs/blob/88403af672427f61cad73a91ec9b7df470912ea4/docs/core/tools/dotnet-new-details.md) | [dotnet new details](https://review.learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new-details?branch=pr-en-us-36528) |
| [docs/csharp/how-to/parse-strings-using-split.md](https://github.com/dotnet/docs/blob/88403af672427f61cad73a91ec9b7df470912ea4/docs/csharp/how-to/parse-strings-using-split.md) | [How to separate strings using String.Split in C\#](https://review.learn.microsoft.com/en-us/dotnet/csharp/how-to/parse-strings-using-split?branch=pr-en-us-36528) |
| [docs/csharp/language-reference/compiler-options/advanced.md](https://github.com/dotnet/docs/blob/88403af672427f61cad73a91ec9b7df470912ea4/docs/csharp/language-reference/compiler-options/advanced.md) | [Advanced C# compiler options](https://review.learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/advanced?branch=pr-en-us-36528) |
| [docs/csharp/language-reference/keywords/let-clause.md](https://github.com/dotnet/docs/blob/88403af672427f61cad73a91ec9b7df470912ea4/docs/csharp/language-reference/keywords/let-clause.md) | ["let clause - C# Reference"](https://review.learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/let-clause?branch=pr-en-us-36528) |
| [docs/csharp/language-reference/keywords/query-keywords.md](https://github.com/dotnet/docs/blob/88403af672427f61cad73a91ec9b7df470912ea4/docs/csharp/language-reference/keywords/query-keywords.md) | [Query keywords (C# Reference)](https://review.learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/query-keywords?branch=pr-en-us-36528) |
| [docs/csharp/linq/perform-inner-joins.md](https://github.com/dotnet/docs/blob/88403af672427f61cad73a91ec9b7df470912ea4/docs/csharp/linq/perform-inner-joins.md) | [docs/csharp/linq/perform-inner-joins](https://review.learn.microsoft.com/en-us/dotnet/csharp/linq/perform-inner-joins?branch=pr-en-us-36528) |
| [docs/csharp/programming-guide/statements-expressions-operators/expression-bodied-members.md](https://github.com/dotnet/docs/blob/88403af672427f61cad73a91ec9b7df470912ea4/docs/csharp/programming-guide/statements-expressions-operators/expression-bodied-members.md) | [Expression-bodied members (C# programming guide)](https://review.learn.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members?branch=pr-en-us-36528) |

</details>

### CODE REVIEW

**Review by jsquire — APPROVED** [review]
SDK-related changes look good to me.

File: docs/core/tools/custom-templates.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ An easy way to exclude all code files from being compiled by your template proje
 
 An easy way to structure your template pack is to put all templates in individual folders, and then each template folder inside of a *templates* folder that is located in the same directory as your *.csproj* file. This way, you can use a single project item to include all files and folders in the *templates* as **content**. Inside of an `<ItemGroup>` element, create a `<Content Include="templates\**\*" Exclude="templates\**\bin\**;templates\**\obj\**" />` item.
 
-Here's an example *.csproj* file that follows all of the guidelines above. It packs the *templates* child folder to the *content* package folder and excludes any code file from being compiled.
+Here's an example *.csproj* file that follows all of the previous guidelines. It packs the *templates* child folder to the *content* package folder and excludes any code file from being compiled.
```
Comment (tdykstra) [inline docs/core/tools/custom-templates.md:177]:
```suggestion
Here's an example *.csproj* file that follows all of these guidelines. It packs the *templates* child folder to the *content* package folder and excludes any code file from being compiled.
```

File: docs/core/tools/dotnet-new-details.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ If the package is installed locally or is found on the official NuGet website, i
 
 - **`-h|--help`**
 
-  Prints out help for the search command.
+  Prints help for the search command.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-new-details.md:47]:
```suggestion
  Displays help for the search command.
```

**Review by tdykstra — APPROVED** [review]
LGTM, a couple of suggestions

**Pull request merged**